test/Driver/batch_mode_with_supplementary_filelist.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %empty-directory(%t)
+// RUN: echo 'print("Hello, World!")' >%t/main.swift
+// RUN: touch %t/file-01.swift %t/file-02.swift %t/file-03.swift
+//
+// Ensure that the supplementary output filelist argument is passed to the frontend:
+//
+// RUN: %swiftc_driver -enable-batch-mode -driver-use-filelists -j2 %t/main.swift %t/file-01.swift %t/file-02.swift %t/file-03.swift -o %t/file-01.o  -o %t/file-02.o  -o %t/file-03.o -### | %FileCheck %s -check-prefix=CHECK-SUPPLEMENTARY-OUTPUT-FILELIST
+//
+// CHECK-SUPPLEMENTARY-OUTPUT-FILELIST: -supplementary-output-filemap {{.*}}/supplementaryOutputs-
+//
+// RUN: %swiftc_driver -enable-batch-mode -driver-use-filelists -j2 %t/main.swift  %t/file-01.swift %t/file-02.swift %t/file-03.swift  -o %t/a.out
+// RUN: %t/a.out | %FileCheck %s -check-prefix=CHECK-HELLO-WORLD
+// CHECK-HELLO-WORLD: Hello, World!
```

Comment:
Please split the executable part of this test into a separate file, so we can mark it with `// REQUIRES: executable_test`. You'll also have to use `%target-build-swift` instead of `%swiftc_driver` to get the right build settings, and `%target-run` to make sure the resulting executable is run correctly for non-host platforms like iOS.

Reply:
Thanks. I'll try to study Testing.md.

Reply:
Did split up the test as suggested.